### DISCUSSION TITLE
Plex Library linked to multiple folders and 'arr instances

### DISCUSSION THREAD

**Discussion by DataLoreQ** [meisnate12/Plex-Meta-Manager#2032] [Q&A]
I have a Plex library named **Movies**, which has folders under it named: 
/media/movies
/media/foreign.movies
/media/movies.editions

I also have two Radarr instances (movies & movie.editions) for those folders.

**movies**
/movies/movies
/movies/foreign.movies

**movie.editions**
/movies/movies.editions

This is duplicated for my 4K movies.

How can I configure Kometa to work with this?  And more information is needed, please let me know.

I have found that PMM/Kometa will upgrade my "special editions" if I have them under one radarr.

Thank you!

closest I could find similar was this discussion: https://github.com/Kometa-Team/Kometa/discussions/989

**Comment by chazlarson** [comment]
What do you mean by "work with this"?

Generally, Kometa doesn't know or care about the filesystem behind your libraries.

If you want to decide between some number of Radarr instances and/or root directories for a given library [Movie A gets `/movies/movies`, Movie B gets `/media/movies.editions`, Movie C needs to go to the 4K radarr], you will need to set things up so that in a specific context from which Kometa is sending a thing to Radarr, there is only one root dir that applies, for example perhaps at the collection level.  There is no way to make that decision on a movie-by-movie basis at runtime.

> I have found that PMM/Kometa will upgrade my "special editions" if I have them under one radarr.

I have no idea what this means.  If you can provide a log showing this happening that would be helpful.

Generally, this sort of thing is better handled on the Discord, as there are more folks in there.  There are only a couple of us that actually look here.

**Reply by DataLoreQ** [reply]
Thanks, I'll post it on Discord then (yes I am on it).

For Radarr #1, the config Path:/movies is "/mnt/user/data/media/",
Then, the "/movies/movies" and "/movies/foreign.movies" are in the root folders for Radarr #1.

For Radarr #2, the config Path:/movies is "/mnt/user/data/media/",
Then, the "/movies/movies.editions" is in the root folders for Radarr #2.

Plex has Path: /media: as "/mnt/user/data/media/"
Then the Movies library has the folders: "/media/movies, /media/foreign.movies, and /meda/movies.editions.

As for upgrading the "special editions", I have had it delete my special edition for a non-special edition movie.  I'll have to check my log files to see if it still exists.

Either way, than you for your response, and I'll get on Discord after I'm back running a friend around!

Thank you again!
